helm/helm-deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+helm upgrade ${CHART_NAME} \
+ ./helm/${CHART_NAME} \
+--install \
+--reset-values \
+--timeout 10m \
+--history-max 3 \
+--namespace ${KUBE_NAMESPACE} \
+--set version=${VERSION} $*
```

Comment:
You won't need the $* here if you plan to run this manually
```suggestion
--set version=${VERSION}
```